packages/siri-sx-generator/test/siriSXUploadToDynamo.ts

```diff
@@ -3,6 +3,8 @@ import { BatchWriteCommand } from "@aws-sdk/lib-dynamodb";
 import { randomUUID } from "crypto";
 import { baseDisruptionJson } from "./testData";
 import { getDdbDocumentClient } from "../util/awsClient";
+import { PtSituationElement } from "@create-disruptions-data/shared-ts/siriTypes";
```

Comment:
some linting warnings about order of imports

Reply:
All checks are passing currently

Reply:
Ah - I see what you mean. Will fix.